demo/demo-crossapp/crossapp-client/src/main/java/org/apache/servicecomb/demo/crossapp/CrossappClient.java

```diff
@@ -32,12 +40,14 @@
   private static HelloWorld helloWorld;
 
   public static void main(String[] args) throws Exception {
+    System.setProperty("sun.net.http.allowRestrictedHeaders", "true");
```

Comment:
a little strange
crossapp demo is to verify cross ServiceComb application

but no problem, we will merge most demo and integration test to one or two project in the future.